### PR TITLE
Fix SSL handshake with Paypal Sandbox

### PR DESCRIPTION
**Pull request by Boubalou**
## 📖 Description

We had a major issue with the SSL handshake coming from the Paypal Sandbox environment which led us to think that our library could also fail in production. 

## 👷 Work done

Add new certificats of thrust for Paypal Sandbox and beyond...

Took the cacert.pem from ActiveMerchant which is a very popular library from which we can assume that all certificats bundled in there are _safe_.

## 🗒 Note

Also refactored the Travis file to make sure it actually build and test the real thing.

## 🎉 Result

| Before | After |
| ------ | ------ |
| Screen Shot 2019-04-12 at 14 33 42 | Screen Shot 2019-04-12 at 14 33 30 |

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+
 rvm:
-  - "2.0.0"
-  - "1.9.3"
-  - "1.8.7"
-notifications:
-  recipients:
-    - DL-PP-Platform-Ruby-SDK@ebay.com
+  - '2.3.8'
+
```
Comment (remiprev) [inline .travis.yml]: Il manque un newline 😄

**Comment by Boubalou** [comment]
Special mention to @jfcartkeep for the help on pinpointing the possible issue.

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+# Run RSpec examples
+script:
+  - 'echo "Running specs" && RAILS_ENV=test bundle exec rspec'
+
```
Comment (simonprev) [inline .travis.yml]: The line break should be included in this file

File: .gitignore [hunk 3]
```diff
@@ -3,7 +3,6 @@
 .bundle
 .config
 .yardoc
-Gemfile.lock
```
Comment (jfcartkeep) [inline .gitignore]: I don't think we should push `Gemfile.lock` in the repo (see [this](https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/))

File: spec/button_manager_spec.rb [hunk 4]
```diff
@@ -16,7 +16,7 @@
   end
 
   it "Validate ipn message" do
-    @api.ipn_valid?("Invalid").should be_false
```
Comment (jfcartkeep) [inline spec/button_manager_spec.rb:19]: This is needed?

**Review by jfcartkeep — APPROVED** [review]

File: spec/button_manager_spec.rb (hunk 4 above)
Comment (Boubalou) [reply spec/button_manager_spec.rb:19]: Yes! The tests where failing otherwise. (unmaintained tests, I guess.)

File: .gitignore (hunk 3 above)
Comment (Boubalou) [reply .gitignore]: Interesting, I did that to prevent errors on Travis, which led me to think that I had to commit it. Hmmm.

**Pull request merged**